test/Project.toml

```diff
@@ -0,0 +1,8 @@
+[deps]
```

Comment:
Are you sure this is the recommended approach (again, based on Example.jl)?

Reply:
https://julialang.github.io/Pkg.jl/dev/creating-packages/#Test-specific-dependencies-in-Julia-1.2-and-above-1 It is in the Pkg.jl documentation, but for Julia 1.2 onward.

Reply:
Interesting. Though this approach doesn't work on Julia 1.0 AFAICT, so we need to stick with the old one for now.

Reply:
Aye. Ideally we could do some Travis logic to support the current system when Julia is the LTS à la Project/REQUIRE.